MANUAL.md

```diff
@@ -155,7 +155,7 @@ are all optional:
 
 The `NAME` option is always used, by default. Specifying `/NAME/` without a
 process name will use a system generated name, as would not specifying the
-option at all. `/NAME $pn1` would create a process with the name `$pn` instead
+option at all. `/NAME $pn1/` would create a process with the name `$pn` instead
```

Comment:
Strange, I though I had seen that slash before. Professional Blindness I guess...